internal/app/subsystems/api/grpc/grpc.go

```diff
@@ -33,6 +33,7 @@ func New(api api.API, config *Config) api.Subsystem {
 
 	server := grpc.NewServer(grpc.UnaryInterceptor(s.log)) // nosemgrep
 	grpcApi.RegisterPromiseServiceServer(server, s)
```

Comment:
Not required for this PR, but I wonder how the name became `PromiseServiceServer`? Would be nice to align on:
- PromiseServer
- SchedulesServer

Reply:
it became promiseservice because that is the name of the service "PromiseService" in the proto. I can just change it to service "Promise" and will be the same.

Reply:
it's PromiseService because the type promise is already defined in proto.

Reply:
Howcome that's not the case for schedule as well? We have a "Schedule" in the schemas there too